web/viewer.html

```diff
@@ -263,30 +263,36 @@
                 <span id="numPages" class="toolbarLabel"></span>
               </div>
               <div id="toolbarViewerRight">
-                <button id="presentationMode" class="toolbarButton hiddenLargeView" title="Switch to Presentation Mode" tabindex="31" data-l10n-id="presentation_mode">
+                <div id="editionModeButtons" class="splitToolbarButton hidden" role="radiogroup">
+                  <button id="editorFreeText" class="toolbarButton" title="Add some free text" role="radio" aria-checked="false" tabindex="31" data-l10n-id="editor_free_text">
```

Comment:
When a mode is enabled, here FreeText, the button is in a toggled state:
![image](https://user-images.githubusercontent.com/5641725/171904485-055d8975-2b5d-447f-98da-4f5ad96bc0bd.png)

and when it's clicked, then it switches to untoggled state and then no more edition is possible.
It is the way it works in Edge (I don't pretend that Edge is a model to follow: I don't have any strong opinions on this kind of stuff and I hope some UX people could help on this kind of stuff).

Anyway, could we considering doing that in a follow-up ?

Reply:
> and when it's clicked, then it switches to untoggled state and then no more edition is possible.

As soon as you add support for at least one more kind of Annotation, this will start becoming really confusing as far as I'm concerned.
Hence why I think that we should fix this *now*, since it will simplify future changes and would also be consistent with all of our other buttons that control related state (see e.g. the Sidebar buttons, or Scroll/Spread-mode buttons).

Reply:
One thing that's not particularly nice, in my opinion, about the current implementation is that it makes the `BaseViewer`-setter look inconsistent/strange when compared to many of the other ones (since they purposely ignores the "value is the same"-case).

Reply:
Ok, as said I don't have any opinion on UX stuff: usually I rely on people who know.... and you're definitely a better expert than me.
Any idea for an icon ?

Reply:
I'm not an expert on UX matters either, but I just want to try and make it as easily as possible to work with this w.r.t. any future changes (since I assume INK-annotations may be next in line here, given e.g. the code comments in `BaseViewer`).

Assuming that the icons are indeed only *temporary*, and given that all of this is disabled by default, how about maybe adding a copy of the `selectTool` here?

Reply:
The ink stuff is ready ;)
Ok for selectTool.